### PR TITLE
Fix issue #1253

### PR DESCRIPTION
**Pull request by gmloose**
Fixex issue #1253. 

`Path::absolutName()` no longer strips off the leading dot of so-called dot-files -- files or directories whose name start with a `.`.

### CODE REVIEW

File: casa/OS/Path.cc [hunk 1]
```diff
@@ -395,23 +395,12 @@ String Path::makeAbsoluteName (const String& inString) const
...
-                case '.' :
-                   from = 0;
-                   break;
-               }
-	    } 
-	    workString = workString.from (from);
-	}
+    if (workString == ".") {
+        workString = "";
+    } else if (workString.before(2) == "./") {
```
Comment (mordante) [inline casa/OS/Path.cc:402]:
```suggestion
    } else if (workString.startsWith("./")) {
```
For readability I would change this, especially since `std::string::starts_with` exists too.

File: casa/OS/Path.cc [hunk 2]
```diff
@@ -395,23 +395,12 @@ String Path::makeAbsoluteName (const String& inString) const
 	return inString;
     }
     // Otherwise we have a relative pathname.
-    // Remove a possible leading . or ./
+    // Remove a possible leading ./
```
Comment (mordante) [inline casa/OS/Path.cc:398]: I would reword this since it partly describes the code but not completely.
No real suggestion, but removing this line would be fine by me. The line above would be enough for me.

**Review by mordante — COMMENTED** [review]
Thanks for the fixes, some comments, but the general approach looks good.

File: casa/OS/Path.cc (hunk 2 above)
Comment (gmloose) [reply casa/OS/Path.cc]: Changed as suggested.

**Comment by mordante** [comment]
LGTM, please give @gervandiepen some time to have a second look too.

**Review by mordante — APPROVED** [review]
